src/Arcus.Messaging.Tests.Workers/MessageHandlers/OrdersMessagePump.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Arcus.EventGrid.Publishing;
+using Arcus.EventGrid.Publishing.Interfaces;
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Pumps.ServiceBus;
+using Arcus.Messaging.Tests.Core.Events.v1;
+using Arcus.Messaging.Tests.Core.Messages.v1;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.Messaging.Tests.Workers.MessageHandlers
+{
+    public class OrdersMessagePump : AzureServiceBusMessagePump<Order>
+    {
+        private readonly IEventGridPublisher _eventGridPublisher;
+
+        public OrdersMessagePump(IConfiguration configuration, IServiceProvider serviceProvider, ILogger<OrdersMessagePump> logger)
+            : base(configuration, serviceProvider, logger)
+        {
+            var eventGridTopic = configuration.GetValue<string>("EVENTGRID_TOPIC_URI");
+            var eventGridKey = configuration.GetValue<string>("EVENTGRID_AUTH_KEY");
+
+            _eventGridPublisher = EventGridPublisherBuilder
```

Comment:
any reason why this isn't injected in the ctor?

Reply:
Because it's for testing and doesn't really matter imo but if you really want I can move it if you think it should.

Reply:
Updated it anyway.